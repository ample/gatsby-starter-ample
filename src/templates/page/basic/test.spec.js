import React from "react"
import renderer from "react-test-renderer"

import { template as BasicPage, fixtures } from "."

describe("BasicPage", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<BasicPage {...fixtures.default} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
