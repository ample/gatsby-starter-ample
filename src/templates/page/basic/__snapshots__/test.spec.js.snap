// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BasicPage renders correctly 1`] = `
Array [
  <header
    className="header"
  >
    <div>
      <div
        className="logo_container"
      >
        <a
          className="logo"
          href="/"
        >
          <svg
            viewBox="0 0 292.642 340.529"
          >
            <g>
              <g>
                <path
                  d="M183.017 315.94c-3.61-15.747 6.689-33.17 25.385-42.987 18.688-9.808 71.15-37.94 71.15-37.94l6.924 30.535c7.247 31.714-15.002 64.002-49.693 72.163-21.475 5.048-47.37 6.178-53.766-21.771zM111.27 323.753a91.566 91.566 0 005.675 16.776 157.222 157.222 0 014.916-305.155L275.019 0l17.623 76.35c-27.351-16.405-63.045-22.577-99.318-14.165-49.635 11.526-86.344 46.89-96.898 87.81a37.41 37.41 0 00-.275 17.44 38.338 38.338 0 0046 28.69c36.188-8.401 18.127-57.647 66.273-68.83 32.638-7.577 49.863 11.45 52.235 21.703l3.484 14.981-91.261 43.758a118.729 118.729 0 00-34.66 24.665c-23.498 24.543-34.659 58.132-26.953 91.351z"
                  fill="#0065f2"
                />
              </g>
            </g>
          </svg>
        </a>
        <button
          className="menu_button"
          onClick={[Function]}
        >
          <span>
            Menu
          </span>
          <svg
            height={21}
            viewBox="0 0 448 512"
            width={26}
          >
            <path
              d="M442 114H6a6 6 0 01-6-6V84a6 6 0 016-6h436a6 6 0 016 6v24a6 6 0 01-6 6zm0 160H6a6 6 0 01-6-6v-24a6 6 0 016-6h436a6 6 0 016 6v24a6 6 0 01-6 6zm0 160H6a6 6 0 01-6-6v-24a6 6 0 016-6h436a6 6 0 016 6v24a6 6 0 01-6 6z"
            />
          </svg>
        </button>
      </div>
      <div
        className="navigation_container"
      >
        <ul
          className="navigation top_navigation"
        >
          <li>
            <a
              href="#"
              onClick={[Function]}
              rel="noopener"
              target="_blank"
            >
              Duis
            </a>
          </li>
          <li>
            <a
              href="#"
              onClick={[Function]}
              rel="noopener"
              target="_blank"
            >
              Cupidatat
            </a>
          </li>
          <li>
            <a
              href="#"
              onClick={[Function]}
              rel="noopener"
              target="_blank"
            >
              Ullamco
            </a>
          </li>
          <li>
            <a
              href="#"
              onClick={[Function]}
              rel="noopener"
              target="_blank"
            >
              Labore
            </a>
          </li>
          <li>
            <a
              href="#"
              onClick={[Function]}
              rel="noopener"
              target="_blank"
            >
              Consectetur
            </a>
          </li>
          <li>
            <a
              href="#"
              onClick={[Function]}
              rel="noopener"
              target="_blank"
            >
              Pariatur
            </a>
          </li>
        </ul>
        <ul
          className="navigation main_navigation"
        >
          <li>
            <a
              href="#"
              onClick={[Function]}
              rel="noopener"
              target="_blank"
            >
              Reprehenderit
            </a>
          </li>
          <li>
            <a
              href="#"
              onClick={[Function]}
              rel="noopener"
              target="_blank"
            >
              Officia sint
            </a>
          </li>
          <li>
            <a
              href="#"
              onClick={[Function]}
              rel="noopener"
              target="_blank"
            >
              Veniam
            </a>
          </li>
          <li>
            <a
              href="#"
              onClick={[Function]}
              rel="noopener"
              target="_blank"
            >
              Consequat
            </a>
          </li>
          <li>
            <span
              className="dropdown"
              onClick={[Function]}
              onKeyPress={[Function]}
              role="button"
              tabIndex={0}
            >
              <div
                className="dropdown_trigger"
              >
                Suspendisse semper
                <svg
                  viewBox="0 0 256 512"
                >
                  <path
                    d="M119.5 326.9L3.5 209.1c-4.7-4.7-4.7-12.3 0-17l7.1-7.1c4.7-4.7 12.3-4.7 17 0L128 287.3l100.4-102.2c4.7-4.7 12.3-4.7 17 0l7.1 7.1c4.7 4.7 4.7 12.3 0 17L136.5 327c-4.7 4.6-12.3 4.6-17-.1z"
                  />
                </svg>
              </div>
              <ul
                className="dropdown_menu"
              >
                <li>
                  <a
                    className="link"
                    href="/"
                    onClick={[Function]}
                  >
                    Vestibulum vitae
                  </a>
                </li>
                <li>
                  <a
                    className="link is_external"
                    href="#"
                    onClick={[Function]}
                    rel="noopener"
                    target="_blank"
                  >
                    Pellentesque tempor venenatis
                  </a>
                </li>
                <li>
                  <a
                    className="link is_external"
                    href="#"
                    onClick={[Function]}
                    rel="noopener"
                    target="_blank"
                  >
                    Fusce molestie
                  </a>
                </li>
                <li>
                  <a
                    className="link is_external"
                    href="#"
                    onClick={[Function]}
                    rel="noopener"
                    target="_blank"
                  >
                    Etiam in lacinia dolor
                  </a>
                </li>
              </ul>
            </span>
          </li>
          <li
            className="nav_button"
          >
            <a
              className="button button theme_default"
              href="#"
              onClick={[Function]}
              rel="noopener"
              target="_blank"
            >
              Quisque interdum
            </a>
          </li>
        </ul>
      </div>
    </div>
  </header>,
  <main
    className="basic_page"
  >
    <div
      dangerouslySetInnerHTML={
        Object {
          "__html": "<p>Hello World</p>",
        }
      }
    />
  </main>,
  <footer
    className="footer"
  >
    <div
      className="footer_container"
    >
      <div>
        <div
          className="logo"
        >
          <a
            className="link"
            href="/"
          >
            <svg
              viewBox="0 0 292.642 340.529"
            >
              <g>
                <g>
                  <path
                    d="M183.017 315.94c-3.61-15.747 6.689-33.17 25.385-42.987 18.688-9.808 71.15-37.94 71.15-37.94l6.924 30.535c7.247 31.714-15.002 64.002-49.693 72.163-21.475 5.048-47.37 6.178-53.766-21.771zM111.27 323.753a91.566 91.566 0 005.675 16.776 157.222 157.222 0 014.916-305.155L275.019 0l17.623 76.35c-27.351-16.405-63.045-22.577-99.318-14.165-49.635 11.526-86.344 46.89-96.898 87.81a37.41 37.41 0 00-.275 17.44 38.338 38.338 0 0046 28.69c36.188-8.401 18.127-57.647 66.273-68.83 32.638-7.577 49.863 11.45 52.235 21.703l3.484 14.981-91.261 43.758a118.729 118.729 0 00-34.66 24.665c-23.498 24.543-34.659 58.132-26.953 91.351z"
                    fill="#0065f2"
                  />
                </g>
              </g>
            </svg>
          </a>
        </div>
        <nav
          className="menus"
        >
          <ul
            className="link_list menu"
          >
            <li>
              <strong>
                Quisque
              </strong>
            </li>
            <li>
              <a
                href="#"
                rel="noopener"
                target="_blank"
              >
                Vestibulum vitae
              </a>
            </li>
            <li>
              <a
                href="#"
                rel="noopener"
                target="_blank"
              >
                Pellentesque tempor venenatis
              </a>
            </li>
            <li>
              <a
                href="#"
                rel="noopener"
                target="_blank"
              >
                Fusce molestie
              </a>
            </li>
            <li>
              <a
                href="#"
                rel="noopener"
                target="_blank"
              >
                Etiam in lacinia dolor
              </a>
            </li>
          </ul>
          <ul
            className="link_list menu"
          >
            <li>
              <strong>
                Vestibulum
              </strong>
            </li>
            <li>
              <a
                href="#"
                rel="noopener"
                target="_blank"
              >
                Fusce molestie
              </a>
            </li>
            <li>
              <a
                href="#"
                rel="noopener"
                target="_blank"
              >
                Pellentesque venenatis
              </a>
            </li>
            <li>
              <a
                href="#"
                rel="noopener"
                target="_blank"
              >
                Vestibulum vitae
              </a>
            </li>
            <li>
              <a
                href="#"
                rel="noopener"
                target="_blank"
              >
                lacinia dolor
              </a>
            </li>
          </ul>
        </nav>
      </div>
      <div>
        <nav
          className="social_nav"
        >
          <a
            className="link is_external"
            href="#"
            rel="noopener"
            target="_blank"
          >
            <svg
              viewBox="0 0 512 512"
            >
              <path
                d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
              />
            </svg>
          </a>
          <a
            className="link is_external"
            href="#"
            rel="noopener"
            target="_blank"
          >
            <svg
              viewBox="0 0 320 512"
            >
              <path
                d="M279.14 288l14.22-92.66h-88.91v-60.13c0-25.35 12.42-50.06 52.24-50.06h40.42V6.26S260.43 0 225.36 0c-73.22 0-121.08 44.38-121.08 124.72v70.62H22.89V288h81.39v224h100.17V288z"
              />
            </svg>
          </a>
        </nav>
        <ul
          className="link_list policy_links"
        >
          <li>
            <a
              href="#"
              rel="noopener"
              target="_blank"
            >
              Privacy Policy
            </a>
          </li>
          <li>
            <a
              href="#"
              rel="noopener"
              target="_blank"
            >
              Terms of Service
            </a>
          </li>
        </ul>
        <small
          className="copyright"
        >
          © 
          2020, All Rights Reserved
        </small>
      </div>
    </div>
  </footer>,
]
`;
